cabal.project

```diff
@@ -99,7 +99,7 @@ source-repository-package
   type: git
   location: https://github.com/input-output-hk/cardano-node
   tag: 9a7331cce5e8bc0ea9c6bfa1c28773f4c5a7000f
-  --sha256: 1d9w9119sifi6y2fvlb7cayvv7chv4skyagnnljh6ihqk2d35jrs
+  --sha256: 1scffi7igv4kj93bpjf8yibcaq0sskfikmm00f7r6q031l53y50c
```

Comment:
I will fix that on the `release/9.0.x` branch.

Reply:
That is now fixed on the `release/9.0.x` branch. You should drop this commit and then rebase agasinst `release/9.0.x`.